src/main/java/emissary/output/DropOffUtil.java

```diff
@@ -1086,7 +1086,7 @@ public void processMetadata(final List<IBaseDataObject> payloadList) {
                 final List<IBaseDataObject> childObjList = p.getExtractedRecords();
                 Collections.sort(childObjList, new emissary.util.ShortNameComparator());
                 for (final IBaseDataObject child : childObjList) {
-                    final int level2 = child.shortName().split("-att-").length;
+                    final int level2 = StringUtils.countMatches(child.shortName(), emissary.core.Family.SEP) + 1;
                     final int parentLevel = level2 - 1;
```

Comment:
Can these two lines be combined as:

`final int parentLevel = StringUtils.countMatches(child.shortName(), emissary.core.Family.SEP);`

Reply:
Good catch. Updated.